src/relax/op/tensor/manipulate.cc

```diff
@@ -391,6 +391,61 @@ TVM_REGISTER_OP("relax.flatten")
     .add_argument("x", "Tensor", "The input tensor.")
     .set_attr<FInferStructInfo>("FInferStructInfo", InferStructInfoFlatten);
 
+/* relax.layout_transform */
+TVM_REGISTER_NODE_TYPE(LayoutTransformAttrs);
+
+Expr layout_transform(Expr x, tir::IndexMap index_map, Optional<PrimValue> pad_value) {
+  ObjectPtr<LayoutTransformAttrs> attrs = make_object<LayoutTransformAttrs>();
+  attrs->index_map = std::move(index_map);
+  attrs->pad_value = std::move(pad_value);
+
+  static const Op& op = Op::Get("relax.layout_transform");
+  return Call(op, {std::move(x)}, Attrs{attrs}, {});
+}
+
+TVM_REGISTER_GLOBAL("relax.op.layout_transform")
+    .set_body_typed([](Expr x, tir::IndexMap index_map, Optional<PrimValue> pad_value) {
+      return layout_transform(x, index_map, pad_value);
+    });
```

Comment:
I suppose we can write like this to eliminate the indirection here.
```suggestion
TVM_REGISTER_GLOBAL("relax.op.layout_transform").set_body_typed(layout_transform);
```